sdk/src/syscall/slothashes.rs

```diff
@@ -0,0 +1,54 @@
+//! named accounts for synthesized data accounts for bank state, etc.
+//!
+//! this account carries the Bank's most recent blockhashes for some N parents
+//!
+use crate::account::Account;
+use crate::hash::Hash;
+use crate::pubkey::Pubkey;
+
+/// "Sysca11SlotHashes11111111111111111111111111"
```

Comment:
@rob-solana , why the ones in "Sysca11" here and above, instead of `l`s? Just to be cute?

Reply:
Base58 has no `l` :-/

Reply:
Oh, duh!